frontend/apps/reader/modules/readermenu.lua

```diff
@@ -241,7 +240,7 @@ function ReaderMenu:onShowReaderMenu(tab_index)
     }
 
     local main_menu
-    if Device:isTouchDevice() then
+    if Device:hasKeys() then
```

Comment:
There are devices which are TouchDevices and don't have keys, actually most :)
With this (I guess), when i don't get my touch menu, but a new windows i have never seen :) "Document menu".